mesop/version.py

```diff
@@ -1,6 +1,6 @@
 """Contains the version string."""
 
-VERSION = "0.12.9"
+VERSION = "0.12.10beta5"
```

Comment:
Is this change necessary?

Reply:
let me take it out. I'll cut a new release soon